test/psim/BUILD

```diff
@@ -0,0 +1,6 @@
+load("//:tools/psim.bzl", "psim_cc_test")
```

Comment:
Maybe you already did this but I missed it in the ocean of files:
- I think it's very essential that we document all testable and buildable targets somewhere. Definitely should do it as part of this PR.

Reply:
Also should list any "getting started" steps in the README, e.g. installilng Bazel.

Reply:
Ah yes, correct. I did forget to add in the getting started information to the README.